mLRS/Common/fail.h

```diff
@@ -102,6 +101,7 @@ void fail(tSerialBase* dbg, uint8_t led_pattern, const char* msg)
 
     led_red_on();
     led_green_on();
+#endif
```

Comment:
should go after the last while(1) (will never be reached)

Reply:
Addressed